Jinja2 test failing

Jinja2 instrumentation tests are failing:
https://github.com/open-telemetry/opentelemetry-python/runs/5024203734?check_suite_focus=true

```
py36-test-instrumentation-jinja2: commands failed

=================================== FAILURES ===================================
__________________ TestJinja2Instrumentor.test_file_template ___________________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_file_template>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
_____________ TestJinja2Instrumentor.test_file_template_with_root ______________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_file_template_with_root>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
_____________ TestJinja2Instrumentor.test_generate_inline_template _____________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_generate_inline_template>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
________ TestJinja2Instrumentor.test_generate_inline_template_with_root ________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_generate_inline_template_with_root>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
______________ TestJinja2Instrumentor.test_render_inline_template ______________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_render_inline_template>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
_________ TestJinja2Instrumentor.test_render_inline_template_with_root _________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_render_inline_template_with_root>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
_______________ TestJinja2Instrumentor.test_render_not_recording _______________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_render_not_recording>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
WARNING  opentelemetry.instrumentation.instrumentor:instrumentor.py:98 Attempting to instrument while already instrumented
__________________ TestJinja2Instrumentor.test_uninstrumented __________________

self = <tests.test_jinja2.TestJinja2Instrumentor testMethod=test_uninstrumented>

    def setUp(self):
        super().setUp()
        Jinja2Instrumentor().instrument()
        # prevent cache effects when using Template('code...')
        if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
            # by clearing functools.lru_cache
>           jinja2.environment.get_spontaneous_environment.clear()
E           AttributeError: 'functools._lru_cache_wrapper' object has no attribute 'clear'

test_jinja2.py:37: AttributeError
------------------------------ Captured log call -------------------------------
```

--- a/instrumentation/opentelemetry-instrumentation-jinja2/tests/test_jinja2.py
+++ b/instrumentation/opentelemetry-instrumentation-jinja2/tests/test_jinja2.py
@@ -37,7 +37,7 @@ class TestJinja2Instrumentor(TestBase):
             jinja2.environment.get_spontaneous_environment.cache_clear()
         else:
             # by clearing jinja2.utils.LRUCache
-            jinja2.environment._spontaneous_environments.cache_clear()  # pylint: disable=no-member
+            jinja2.environment._spontaneous_environments.clear()  # pylint: disable=no-member
 
         self.tracer = get_tracer(__name__)
 

--- a/instrumentation/opentelemetry-instrumentation-jinja2/tests/test_jinja2.py
+++ b/instrumentation/opentelemetry-instrumentation-jinja2/tests/test_jinja2.py
@@ -34,10 +34,10 @@ class TestJinja2Instrumentor(TestBase):
         # prevent cache effects when using Template('code...')
         if version.parse(jinja2.__version__) >= version.parse("3.0.0"):
             # by clearing functools.lru_cache
-            jinja2.environment.get_spontaneous_environment.clear()
+            jinja2.environment.get_spontaneous_environment.cache_clear()
         else:
             # by clearing jinja2.utils.LRUCache
-            jinja2.environment._spontaneous_environments.clear()  # pylint: disable=no-member
+            jinja2.environment._spontaneous_environments.cache_clear()  # pylint: disable=no-member
 
         self.tracer = get_tracer(__name__)
 

